src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+/* блок активних зірок */
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 > * {
+  background-image: url(../images/star-active.svg);
+}
+
+/* блок неактивних зірок */
+
+.stars--0 > *,
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
No need to overcode, without imports in a separate css file, all of this can be contained in one css file it is better to immediately set the background to the class stars__star - star.svg and then change them to the active one.